xicam/gui/bluesky/top_utils.py

```diff
@@ -0,0 +1,31 @@
+from traitlets import TraitType
+from traitlets.config.loader import (PyFileConfigLoader, ConfigFileNotFound,
+                                     Config)
+
+CONFIG_FILE_NAME = 'bluesky_browser_config.py'
```

Comment:
This should probably be updated. In the future we can have a larger discussion about configuration management, not sure if that should hold up this PR.